src/integration/java/io/pinecone/integration/controlPlane/CreateListAndDeleteIndexTest.java

```diff
@@ -37,6 +38,8 @@ public void createAndDelete() throws IOException, InterruptedException {
                 .withMetric("euclidean");
         pinecone.createIndex(request);
 
+        isIndexReady(indexName, pinecone);
```

Comment:
If an index is created, I must check if it is ready before any other data or control plane operations.